public_html/wp-content/plugins/pattern-creator/view/editor.php

```diff
@@ -10,35 +10,50 @@
 
 // @todo Permissions TBD, see https://github.com/WordPress/pattern-directory/issues/30.
```

Comment:
Can you remove this todo comment before merging?

Reply:
Done in 7099169